src/python/tests/core/crash_analysis/stack_parsing/stack_analyzer_data/golang_sigsegv_panic.txt

```diff
@@ -0,0 +1,10 @@
+root@943ca8071e8b:/out# ./fuzzer-bzip2 
+panic: runtime error: invalid memory address or nil pointer dereference
+[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x5da137]
+
+goroutine 1 [running]:
+math.glob..func1(0x5da117)
+  /src/go/src/math/exp_asm.go:11 +0x7
+math.init.ializers()
```

Comment:
hmm... is this function "math.init.ializers()" weird on purpose?

Reply:
I think so :D from https://golang.org/doc/go1.12#runtime:

<...> Doing a traceback during the initialization of a global variable will now show a function named `PKG.init.ializers`.

Reply:
ACK